shell/platform/embedder/tests/embedder_gl_unittests.cc

```diff
@@ -3522,17 +3518,15 @@ TEST_F(EmbedderTest, CompositorRenderTargetsAreInStableOrder) {
 }
 
 TEST_F(EmbedderTest, FrameInfoContainsValidWidthAndHeight) {
-  auto& context = GetEmbedderContext(EmbedderTestContextType::kOpenGLContext);
-
-  EmbedderConfigBuilder builder(context);
-  builder.SetOpenGLRendererConfig(SkISize::Make(600, 1024));
-  builder.SetDartEntrypoint("push_frames_over_and_over");
-
+  auto& context = static_cast<EmbedderTestContextGL&>(
+      GetEmbedderContext(EmbedderTestContextType::kOpenGLContext));
```

Comment:
In a followup patch I'm going to try rewriting this method to be templatised so we can do this and just get the right type out automatically like you can do with `std::variant`:
```cc
auto& context GetEmbedderContext<EmbedderTestContextGL>();
```